README.md

```diff
@@ -27,34 +27,33 @@ Sandbox API Repository to describe, develop, document, and test the DedicatedNet
 * Describe, develop, document, and test the APIs
 * Started: October 2024
 
-## Release Information
-
-**Version Control Notice:** This project undergoes active development with frequent updates to the main branch. No backward compatibility is guaranteed for any code in development branches, including main, until an official release is published. Code changes, including potential reversions, may occur between commits. We recommend using the latest official release version.
-
-**NEW**: r1.2 with initial public release of dedicated-network v0.1.0, dedicated-network-profiles v0.1.0 and dedicated-network-accesses v0.1.0 is available.
+<!-- CAMARA:RELEASE-INFO:START -->
+<!-- The following section is automatically maintained by the CAMARA project-administration tooling: https://github.com/camaraproject/project-administration -->
 
-**The latest public release is [r1.2](https://github.com/camaraproject/DedicatedNetworks/releases/tag/r1.2), with the API versions listed below**
-
-To facilitate understanding of these APIs, please consult the [General Description](https://github.com/camaraproject/DedicatedNetworks/blob/r1.2/documentation/API_documentation/DedicatedNetworks_GeneralDescription.md) for essential concepts and workflows prior to examining the technical specifications.
```

Comment:
I would suggest to keep this line, as it acts more as a general introduction to the APIs. Maybe we can move this into the first section, i.e. before Scope. @hdamker Do you have a opinion or suggestion?